NewBank/newbank/server/NewBank.java

```diff
@@ -3,14 +3,17 @@
 import java.util.ArrayList;
 import java.lang.*;
 import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
 
 public class NewBank {
 
     private static final NewBank bank = new NewBank();
 
-    private ArrayList<LocalDate> payments = new ArrayList<>();
     NewBankDatabaseHandler database = new NewBankDatabaseHandler();
 
+    private ArrayList<Microloan> microloans = new ArrayList<>();
```

Comment:
This object is only used in the scope of the loanRequest function, so I think it would be better to declare it there